{"name": "Не удается залогиниться в систему", "status": "passed", "description": "Попытка логина с неправильно введенными или отсутствующими логином или паролем", "parameters": [{"name": "wrong_creds", "value": "{'login': 'qwerty1ytrewq', 'password': ''}"}, {"name": "status_code", "value": "400"}], "start": 1739452729246, "stop": 1739452729581, "uuid": "7ca08d9e-da5d-42b8-bf77-834a3399fa1f", "historyId": "66b025c06898c35280c980a1d3e2fecf", "testCaseId": "01ceaaa13868027d2f21fb2961bdf35d", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_wrong_password_or_login_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "GrayCat"}, {"name": "thread", "value": "3508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}